Example/App.tsx

```diff
@@ -27,8 +27,6 @@ import Modals from './src/screens/Modals';
 import Orientation from './src/screens/Orientation';
 import SearchBar from './src/screens/SearchBar';
 
-enableScreens();
```

Comment:
I think you should also remove the import of the method since it is not used

Reply:
Fixed in af9aec6